components/gitpod-cli/cmd/git-track-command.go

```diff
@@ -76,6 +80,7 @@ var gitTrackCommand = &cobra.Command{
 		log.WithField("command", gitTrackCommandOpts.GitCommand).
 			Info("tracking the GitCommand event")
 
+		// TODO(ak) use segment directly + supervisor info to get workspace and isntance IDs, don't use server
```

Comment:
it is for future, not sure what would be consequence of moving to another source, i.e. from trusted to untrusted.

For Mixpanel there are none, not sure about other reports. cc @jakobhero @loujaybee